pages/requests/create-replacement.tsx

```diff
@@ -0,0 +1,184 @@
+        <GridItem paddingTop="32px">
+          <Box
+            border="1px solid"
+            borderColor="border.secondary"
+            borderRadius="12px"
+            bgColor="white"
+            paddingTop="32px"
+            paddingBottom="40px"
+            paddingX="40px"
+            align="left"
+          >
+            <Text textStyle="display-small-semibold" paddingBottom="20px">
+              {`Permit Holder's Information`}
+            </Text>
+            <PermitHolderInformationForm
+              permitHolderInformation={permitHolderInformation}
+              onChange={setPermitHolderInformation}
+            />
+          </Box>
+        </GridItem>
+        {/* Reason For Replacement Form */}
+        <GridItem paddingTop="32px">
+          <Box
+            border="1px solid"
+            borderColor="border.secondary"
+            borderRadius="12px"
+            bgColor="white"
+            paddingTop="32px"
+            paddingBottom="40px"
+            paddingX="40px"
+            align="left"
+          >
+            <Text textStyle="display-small-semibold" paddingBottom="20px">
+              {`Reason For Replacement Form`}
+            </Text>
+            <ReasonForReplacementForm reasonForReplacement={reasonDetails} onChange={setReason} />
+          </Box>
+        </GridItem>
+        {/* Payment Details Form */}
+        <GridItem paddingTop="32px" paddingBottom="68px">
+          <Box
+            border="1px solid"
+            borderColor="border.secondary"
+            borderRadius="12px"
+            bgColor="white"
+            paddingTop="32px"
+            paddingBottom="40px"
+            paddingX="40px"
+            align="left"
+          >
+            <Text textStyle="display-small-semibold" paddingBottom="20px">
+              {`Payment, Shipping, and Billing Information`}
+            </Text>
+            <PaymentDetailsForm paymentInformation={paymentDetails} onChange={setPaymentDetails} />
+          </Box>
+        </GridItem>
+        {/* Footer */}
+        <Box
+          position="fixed"
+          left="0"
+          bottom="0"
+          right="0"
+          paddingY="20px"
+          paddingX="188px"
+          bgColor="white"
+          boxShadow="dark-lg"
+        >
+          <Stack direction="row" justifyContent="space-between" alignItems="center">
+            <Box>
+              <Text textStyle="body-bold">
+                User ID:
+                <Box as="span" color="primary">
+                  {' '}
+                  {permitHolderID}
+                </Box>
+              </Text>
+            </Box>
+            <Box>
+              <Button
+                bg="background.gray"
+                _hover={{ bg: 'background.grayHover' }}
+                color="black"
+                marginRight="20px"
+                height="48px"
+                width="149px"
+              >
+                <Text textStyle="button-semibold">Cancel</Text>
+              </Button>
```

Comment:
Same here, replace with Next.js `Link`, see https://github.com/uwblueprint/richmond-centre-for-disability/pull/101